### PR TITLE
Fix bug when html is nullable in editor extension

### PR DESCRIPTION
**Pull request by luca-rath**

### CODE REVIEW

**Comment by luca-rath** [comment]
ping @alexander-schranz

File: src/EditorExtension.php [hunk 1]
```diff
@@ -56,28 +56,21 @@ public function getFilters()
...
-     * @return string
      */
-    public function editor(string $html, string $tag = null, string $className = null): string
+    public function editor(?string $html, string $tag = null, string $className = null): string
     {
         $className = $className ?: $this->className;
         $tag = $tag ?: $this->tag;
 
-        return '<' . $tag . ' class="' . $className . '">' . $html . '</' . $tag . '>';
+        return '<' . $tag . ' class="' . $className . '">' . ($html ?? '') . '</' . $tag . '>';
```
Comment (alexander-schranz) [inline src/EditorExtension.php]: I think we should by default omit to render an empty div tag so I would go in both cases with a early return intead:

```
public function editor(?string $html, string $tag = null, string $className = null): string
{
    if (!$html) {
            return '';
     }
```

@luca-rath what do you think?

File: src/EditorExtension.php [hunk 2]
```diff
@@ -56,28 +56,21 @@ public function getFilters()
 
     /**
      * Returns a given html with a wrapped tag and class.
-     *
-     * @param string $html
```
Comment (alexander-schranz) [inline src/EditorExtension.php]: do not remove this we do not omit phpdocs here.

File: src/EditorExtension.php (hunk 1 above)
Comment (luca-rath) [reply src/EditorExtension.php]: I also thought about that, and personally, I would prefer that, but it's not the expected behavior of a developer. If there is `text|editor`, the expected output is always a `div.editor` containing `text`, no matter what `text` is. But maybe that's only my opinion.

Comment (alexander-schranz) [reply src/EditorExtension.php]: @luca-rath I personally would not expect it that it does something with nothing.

**Comment by alexander-schranz** [comment]
@luca-rath Thank you!

**Pull request merged**